configs/models/hf_internlm/lmdeploy_internlm2_chat_7b.py

```diff
@@ -1,15 +1,24 @@
 from opencompass.models import TurboMindModelwithChatTemplate
 
+
 models = [
     dict(
         type=TurboMindModelwithChatTemplate,
-        abbr='internlm2-chat-7b-turbomind',
+        abbr=f'internlm2-chat-7b-lmdeploy',
         path='internlm/internlm2-chat-7b',
-        engine_config=dict(session_len=8192, max_batch_size=16, tp=1),
-        gen_config=dict(top_k=1, temperature=1e-6, top_p=0.9, max_new_tokens=4096),
+        # inference backend of LMDeploy. It can be either 'turbomind' or 'pytorch'.
+        # If the model is not supported by 'turbomind', it will fallback to
+        # 'pytorch'
+        backend='turbomind',
+        # For the detailed engine config and generation config, please refer to
+        # https://github.com/InternLM/lmdeploy/blob/main/lmdeploy/messages.py
+        engine_config=dict(tp=1),
+        gen_config=dict(do_sample=False),
         max_seq_len=8192,
         max_out_len=4096,
-        batch_size=16,
+        # the max number of prompts that LMDeploy receives
+        # in `generate` function
+        batch_size=5000,
```

Comment:
```suggestion
        batch_size=32,
```

I have changed this number into a smaller one. Make the log system update frequently to improve user friendliness.